MultiMystery.py

```diff
@@ -135,6 +137,7 @@ def cancel_remaining(starting_at:int = 0):
 
             errors = []
             dead_or_alive = {}
+            alive = {}
```

Comment:
Why is alive a dict? seems like you're just counting how many survived. Which can either be inferred by asking each task, or using a counter variable.

Reply:
Changed to counter as requested.